themes/custom/droopler_theme/scss/components/_d_p_side_by_side.scss

```diff
@@ -37,23 +37,25 @@ $d-p-side-by-side-image-background-padding-y: 130px !default;
     text-align: center;
   }
 
-  .text-content {
-    padding-bottom: $d-p-side-by-side-padding-y;
+  .side-header {
+    padding: $d-p-side-by-side-padding-y 0;
 
-    .field--name-field-d-main-title {
-      @include font-main-header;
+    .text-content {
 
-      &::after {
-        @include decoration-header-line-main-center($d-p-side-by-side-line-color, $d-p-side-by-side-line-color-inverted);
+      .field--name-field-d-main-title {
+        @include font-main-header;
+
+        &::after {
+          @include decoration-header-line-main-center($d-p-side-by-side-line-color, $d-p-side-by-side-line-color-inverted);
+        }
       }
-    }
 
-    .field--name-field-d-long-text {
-      margin: $d-p-side-by-side-subtitle-margin-y 0;
-      @include font-subtitle;
+      .field--name-field-d-long-text {
+        margin: $d-p-side-by-side-subtitle-margin-y 0;
+        @include font-subtitle;
+      }
```

Comment:
to tez pewnie sie uprosci jak oczyscisz twiga :thinking:  `.side-header` albo `.text-content`  pewnie beda niepotrzebne..